plugins/PrivacyManager/vue/src/Compliance/ComplianceTable.vue

```diff
@@ -0,0 +1,57 @@
+<!--
+  Matomo - free/libre analytics platform
+
+  @link    https://matomo.org
+  @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <table class="card-table dataTable compliance">
+    <thead>
+      <tr>
+        <th class="label">
+          {{ translate('PrivacyManager_ComplianceTableSettingName') }}
+        </th>
+        <th class="label">
+          {{ translate('PrivacyManager_ComplianceTableSettingStatus') }}
+        </th>
+        <th class="label">
+          {{ translate('PrivacyManager_ComplianceTableSettingNotes') }}
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-for="(item, index) in results" :key="index">
+        <td>{{ item.name }}</td>
+        <td :class="{
+          'status': true,
+          'compliant': item.value === 'compliant',
+          'non-compliant': item.value === 'non_compliant',
+          'unknown': item.value === 'unknown',
+        }">
+          <span :class="{
+              'icon-ok': item.value === 'compliant',
```

Comment:
The `icon` class is in the stylesheet, but not used.. let's add it to maintain consistent icons width across the rows, otherwise each icon has slightly different width.

```suggestion
          <span :class="{
              'icon': true,
              'icon-ok': item.value === 'compliant',
```

Reply:
added extra class